src/spikeinterface/qualitymetrics/tests/test_metrics_functions.py

```diff
@@ -384,7 +384,7 @@ def test_calculate_sd_ratio(waveform_extractor_simple):
     sd_ratio = compute_sd_ratio(waveform_extractor_simple)
 
     assert np.all(list(sd_ratio.keys()) == waveform_extractor_simple.unit_ids)
-    assert np.allclose(list(sd_ratio.values()), 1, atol=0.2, rtol=0)
+    assert np.allclose(list(sd_ratio.values()), 1, atol=0.25, rtol=0)
```

Comment:
No it's not meaningful, I think it's due to the recording being relatively short (few spikes). You can increase the tolerance :)

> I don't think you need the list over there, righ?

Maybe not! I'm never sure if the conversion is necessary or not so I tend to always put it because I'm too lazy to check ^^'

Reply:
OK, thanks!